config/deploy.rb

```diff
@@ -6,10 +6,10 @@
 
 set :home_directory, "/home/lyberadmin"
 
-set :branch, 'master'
+#set :branch, 'master'
 
 # Default branch is :master
-# ask :branch, proc { `git rev-parse --abbrev-ref HEAD`.chomp }.call
+ask :branch, proc { `git rev-parse --abbrev-ref HEAD`.chomp }.call
```

Comment:
see above, we could default to master if you'd like, but i like being able to deploy other branches if needed